Born digital config changes in appendix

ioduok opened the pull request:
- changes to some of the action steps
- added rationales for all rows
- edits to name and description of the born digital workflow

ioduok committed:
Born digital config changes in appendix

helrond commented on appendix.md:
```diff
@@ -111,39 +111,39 @@ Legacy born digital materials are those that have been recoverd from digital med
-| Store AIP location                                                              | Store AIP in standard Archivematica directory |  |
-| Upload DIP                                                                       | Do not upload DIP |  |
+| Create SIP(s)                                                                   | Create single SIP and continue processing | Avoid sending packages to temp storage and creating a backlog for processing |
+| Perform file format identification  (Ingest)                                    | No    | No changes have occurred to the transfer packages so the file format identification would be the same, existing data will be used |
```
```suggestion
| Perform file format identification  (Ingest)                                    | No    | No changes have occurred to the transfer packages (such as extraction of packages) so the file format identification would be the same, existing data will be used |
```

helrond reviewed (dismissed):
Just one minor thing, otherwise looks good! Appreciate having these rationales documented!

ioduok committed:
Update appendix.md

more context provided for rationale behind not performing file format identification during Ingest

Co-authored-by: Hillel Arnold <helrond@hotmail.com>
